app/assets/stylesheets/_main.scss

```diff
@@ -0,0 +1,43 @@
+.main {
+  margin: 46px auto 0;
+  width: 325px;
+
+  h1 {
+    font-size: 1.8rem;
+    font-weight: bold;
+    margin: 0 0 1.8rem;
+  }
+
+  h2 {
+    font-size: 1.6rem;
+    font-weight: bold;
+    margin: 0;
+  }
+
+  h3 {
```

Comment:
Not in particular, just experience. A good rule of thumb to start with: if something doesn't need to be specific right now then leave it as a global rule.